deploy/olm-catalog/3scale-operatorsource.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: three-scale-operators
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
```

Comment:
Can you explain me what is this?

Reply:
here? XD

https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md

Reply:
https://github.com/operator-framework/operator-marketplace#operatorsource I found what it means here.

It seems is the URL of Quays' Application Registry:
https://coreos.com/blog/quay-application-registry-for-kubernetes.html